ERROR   2022-09-02 12:33:01,861  D:\lc\code\Software_Test\pytest-auto-api2\logs\error.log:conftest.py:133 异常用例数: 0
ERROR   2022-09-02 12:33:01,863  D:\lc\code\Software_Test\pytest-auto-api2\logs\error.log:conftest.py:134 失败用例数: 0
ERROR   2022-09-02 12:35:56,336  D:\lc\code\Software_Test\pytest-auto-api2\logs\error.log:conftest.py:133 异常用例数: 0
ERROR   2022-09-02 12:35:56,336  D:\lc\code\Software_Test\pytest-auto-api2\logs\error.log:conftest.py:134 失败用例数: 0
